elements/media-quote/demo/index.html

```diff
@@ -19,8 +19,14 @@
       <h3>Basic media-quote demo</h3>
       <demo-snippet>
         <template>
-          <media-quote>
-            This is media-quote
+          <media-quote 
+            src="https://sc-api.psu.edu/s3/files/styles/4_3_1500w/public/2023/04/old-main-blue-sky.jpg?h=08b866d1&itok=ICYTxRXL"
+            alt="Old Main in front of a blue sky"
+            >
+            <span slot="quote">"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Metus vulputate eu scelerisque felis. Nisl nisi scelerisque eu ultrices vitae auctor eu. Id leo in vitae turpis. Eu facilisis sed odio morbi quis commodo."</span>
+            <span slot="author">John Doe</span>
```

Comment:
`div` all the things since span is inline for slot elements here. I'd also make a default one that doesn't require being marked up like the quote one just being any general `div` content. The others look good